assets/app/view/game/corporation.rb

```diff
@@ -87,19 +87,22 @@ def render
           children << h(:div, props, subchildren)
         end
 
-        if @game.status_str(@corporation)
-          props = {
-            style: {
-              grid: '1fr / repeat(2, max-content)',
-              gap: '2rem',
-              justifyContent: 'center',
-              backgroundColor: color_for(:bg2),
-              color: color_for(:font2),
-            },
-          }
+        status_props = {
+          style: {
+            grid: '1fr / repeat(2, max-content)',
+            gap: '2rem',
+            justifyContent: 'center',
+            backgroundColor: color_for(:bg2),
+            color: color_for(:font2),
+          },
+        }
 
-          children << h(:div, props, render_status)
+        if @corporation.trains.any? && !@corporation.floated?
+          if @corporation.trains.any? && !@corporation.floated?
```

Comment:
typo?

Reply:
Yep. Reorganized this section and biffed the cleanup.